### ISSUE TITLE
Item sink priorities not exactly as documented in `dev_zero_guides/managing_items_with_pipes.md`

### ISSUE THREAD

**Issue by Hawk777** [RS485/LogisticsPipes#1703]
According to `dev_zero_guides/managing_items_with_pipes.md`, there are four priorities of destinations:
1. passive suppliers
2. all itemsink modules plus basic pipes
3. terminus modules
4. default routes

However, by experimentation, this does not actually seem to be the case. In fact, an ItemSink module appears to have strictly higher priority than a Mod Based ItemSink Module, so priority (2) is at least two, and maybe more (once you consider all the different kinds of sinks), distinct priorities. This is *good*, because it allows you to establish one storage location for all items from a particular mod *except for* one or two items which you want to keep elsewhere; ItemSinks being higher priority than Mod Based ItemSink Modules means you can simply do the obvious thing (Mod Based ItemSink for the general case plus an ItemSink for the exception) and it works. However, the documentation appears to be wrong.

**Comment by bziemons** [comment]
correct. There are slight differences between the different item sink modules. It probably depends on how they are configured. I'll see if I can get a better explanation after having another look at the code.
